src/cascadia/TerminalSettingsModel/CascadiaSettingsSerialization.cpp

```diff
@@ -1222,15 +1114,12 @@ winrt::hstring CascadiaSettings::DefaultSettingsPath()
     // directory as the exe, that will work for unpackaged scenarios as well. So
     // let's try that.
 
-    HMODULE hModule = GetModuleHandle(nullptr);
-    THROW_LAST_ERROR_IF(hModule == nullptr);
-
     std::wstring exePathString;
-    THROW_IF_FAILED(wil::GetModuleFileNameW(hModule, exePathString));
+    THROW_IF_FAILED(wil::GetModuleFileNameW(nullptr, exePathString));

```

Comment:
`GetModuleFileNameW` docs say that the first parameter can be null to get the filename of the current module.

Reply:
Ah, but the docs state that it will be the filename of the current _executable_ (which may not always be in the same directory as the current module)

However, for terminal this doesn't matter -- all our DLLs are side by side with the EXE.

Reply:
Turns out the rest of our entire code base uses `GetModuleFileNameW` with a `nullptr` parameter. So I'll leave it as it is for consistency.